WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -205,6 +205,18 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	[JsonProperty(PropertyName = "AllowP2trOutputs", DefaultValueHandling = DefaultValueHandling.Populate)]
 	public bool AllowP2trOutputs { get; set; } = true;
 
+	[DefaultValue(false)]
+	[JsonProperty(PropertyName = "AllowP2pkhOutputs", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public bool AllowP2pkhOutputs { get; set; } = false;
+
+	[DefaultValue(false)]
+	[JsonProperty(PropertyName = "AllowP2shOutputs", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public bool AllowP2shOutputs { get; set; } = false;
+
+	[DefaultValue(false)]
+	[JsonProperty(PropertyName = "AllowP2wshOutputs", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public bool AllowP2wshOutputs { get; set; } = false;
+
```

Comment:
We cannot turn them `on` because the clients are broken and they use this info to decide what scripts to during the amount decomposition, this means that wasabi clients would register outputs that they cannot spend.